src/stim/dem/detector_error_model.pybind.cc

```diff
@@ -26,9 +26,8 @@
 #include "stim/simulators/dem_sampler.h"
 
 using namespace stim;
-using namespace stim_pybind;
 
-std::string stim_pybind::detector_error_model_repr(const DetectorErrorModel &self) {
+std::string detector_error_model_repr(const DetectorErrorModel &self) {
```

Comment:
Why did this lose the namespace?

Reply:
at first I wasn't sure if I wanted to put all the aux functions in the namespace, but after a few files realized that that was the way to go

Reply:
fixed